packages/browser.smash

```diff
@@ -148,6 +148,20 @@
         }
     }
 
+    * Click each {{element}} {
+        if(!Array.isArray(element)){
+            const elems = await $(element);
+            elems.forEach(elem => elem.click());
+        }
+        else {
+            const multiElems = element.forEach(elem => elem.click());
+        }
+    }
+
+    * Drag {{dragPosition}} in {{dropPosition}} {
```

Comment:
I would reword this as 
`Drag {{elementFrom}} to {{elementTo}}` and `Drag {{elementFrom}} to {{x}} {{y}}` (2 different functions)

See `dragAndDrop()` docs: https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/lib/input_exports_Actions.html

Reply:
Ok i can reword this and create 2 different functions.